File: 1007 - diferenca/dif.js
var input = require('fs').readFileSync('stdin.txt', 'utf8');
//var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

function diferenca( a, b, c, d ){
    return ((a * b) - (c * d));
}

a = parseInt(lines.shift());
b = parseInt(lines.shift());
c = parseInt(lines.shift());
d = parseInt(lines.shift());

diff = diferenca(a, b, c, d);
console.log('DIFERENCA = '+ diff);

File: 1007 - diferenca/dif.scala
object Main{
    def diferenca( a:Int, b:Int, c:Int, d:Int ) : Int = (a * b) - ( c * d )
    def main( args: Array[String]){
        var a = scala.io.StdIn.readInt() : Int
        var b = scala.io.StdIn.readInt() : Int
        var c = scala.io.StdIn.readInt() : Int
        var d = scala.io.StdIn.readInt() : Int
        var diff = diferenca(a, b, c, d)
        printf("DIFERENCA = %d\n", diff)
    }
}